### DISCUSSION TITLE
空网站，流量监控异常

### DISCUSSION THREAD

**Discussion by geekgao** [1Panel-dev/1Panel#6262] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

Image: https://ibb.co/P9Bmsn7
如图

### 重现步骤

新创建一个虚拟机，然后部署了1Panel

### 期待的正确结果

不太可能有这么大的流量

**Comment by geekgao** [comment]
系统是版本为Ubuntu  24.04

**Comment by ssongliu** [comment]
感谢反馈，可以执行 `ifconfig` 获取比较一下，找到对应的网卡
![image](https://github.com/user-attachments/assets/ab12f10a-fbc2-4cf9-8c44-b38ba1f9e61d)

**Comment by geekgao** [comment]
你好，按照你的方法，我执行了，确实发现是很大的数值。这也有些奇怪。我利用了iftop排查，并没有发现大流量的进程。
这台机器是kvm虚拟化出来的，母机流量也没这么高，况且还是内网环境。 或许不是1Panel的问题。
